packages/perseus/src/widgets/video/video.test.ts

```diff
@@ -49,6 +49,24 @@ describe("video widget", () => {
         expect(container).toMatchSnapshot("first mobile render");
     });
 
+    it("video widget should allow autoplay", () => {
+        // Arrange
+        const apiOptions: APIOptions = {
+            isMobile: false,
+        };
+
+        // Act
+        renderQuestion(question1, apiOptions);
+
+        // Assert
+        // eslint-disable-next-line testing-library/no-node-access
+        expect(
+            document
+                .getElementsByTagName("iframe")[0]
+                .attributes.getNamedItem("allow")?.value,
+        ).toContain("autoplay");
```

Comment:
I agree we need to resort to direct-DOM APIs to find the iframe, but can we use `toHaveAttribute()` instead to do the assertion? 
```suggestion
        expect(
            document.getElementsByTagName("iframe")[0]
        ).toHaveAttribute("allow", "autoplay");
```

Reply:
Good call!